File: app/src/main/java/resource/com/br/santanderapp/model/investmentModel/Info.java
package resource.com.br.santanderapp.model.investmentModel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Info {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("data")
    @Expose
    private String data;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }



}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/Info.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Info: Serializable{

    @SerializedName("name")
    var name: String? = null
    @SerializedName("data")
    var data: String? = null

    override fun toString(): String {
        return "Info(name=$name, data=$data)"
    }
}